requirements-optional.txt

```diff
@@ -16,4 +16,5 @@ pycodestyle==2.5.0
 mypy==0.761
 pydocstyle==5.0.1
 flake8==3.7.9
-pylint==2.4.4
\ No newline at end of file
+pylint==2.4.4
+plotly==4.5.0
```

Comment:
plotly should be moved into requirements.txt, and make sure it's included in setup.py too (with a >= version number)